R/doeResponseSurfaceMethodology.R

```diff
@@ -317,51 +315,56 @@ doeResponseSurfaceMethodology <- function(jaspResults, dataset, options, ...){
     }
   }
 
-
+  set.seed(1)
```

Comment:
is this intended? You can also add an option for a seed in the GUI if you want to ensure reproducibility (there is a QML component for this).

Reply:
Yes, it is intended, I want the random order to always be the same random order, what's the GUI component for it?

Reply:
Its called `SetSeed`